Wingstop_Recovery_Computer_V1/Wingstop_Recovery_Computer_V1.ino

```diff
@@ -389,137 +460,224 @@ void read_barometer(){//takes barometer reading
 }
 
 float read_altitude(){//reads and returns barometer altitude reading
-  while (! bmp.performReading()) {
-    //performs until it gets a valid reading
-  }
-  return bmp.readAltitude(init_pressure); // returns current altitude reading
+  float altitudeReading;
+  do {
+    read_barometer(); 
+    altitudeReading = bmp.readAltitude(init_pressure);
+  } while (abs(altitudeReading - x_current)/(t_current - t_previous) > 1200); 
+  //keeps reading altimiter if speed vertical speed is more than 1200 m/s (should never actually be that fast).
```

Comment:
What is this check for?
typo: altimiter -> altimeter

Reply:
The check is for a reading that is way off from the last reading. I set it up to check the relative difference of the last reading with respect to the time that has passed between the readings (so basically velocity). If a reading is way off from the previous one, it should take the reading again until the velocity would be under 1200 m/s.

Reply:
Fixed the typo.